liveview-android/src/androidTest/java/com/dockyard/liveviewtest/liveview/components/DatePickerShotTest.kt

```diff
@@ -0,0 +1,432 @@
+package com.dockyard.liveviewtest.liveview.components
+
+import android.icu.text.SimpleDateFormat
+import android.icu.util.Calendar
+import android.icu.util.TimeZone
+import androidx.compose.material3.DatePicker
+import androidx.compose.material3.DatePickerDefaults
+import androidx.compose.material3.DateRangePicker
+import androidx.compose.material3.DisplayMode
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Text
+import androidx.compose.material3.rememberDatePickerState
+import androidx.compose.material3.rememberDateRangePickerState
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import com.dockyard.liveviewtest.liveview.util.LiveViewComposableTest
+import org.junit.Test
+import java.util.Date
+
+@OptIn(ExperimentalMaterial3Api::class)
+class DatePickerShotTest : LiveViewComposableTest() {
+    private val jun9th2023 = 1686279600000L
+    private val jun22nd2023 = 1687402800000L
```

Comment:
Not super important but `july9th2023` and `june22nd2023` is nicer to my OCD.

Reply:
It's just a random date that I chose (it has a meaning to me), but I can change ;)

Reply:
Oh I didn't actually have a qualm with the date, just the lack of single letter on the month.

Reply:
Got it... But is "jun" the abbreviation of June? 🤔 I mean... Jan(uary), Feb(ruary), ...